data/azure/instance_types.json

```
@@ -335,111 +335,111 @@
-  "Standard_GS4-4": {"up": "Standard_GS4-8", "down": null, "superseded": {"regular": "Standard_GS4"}},
-  "Standard_GS4-8": {"up": null, "down": "Standard_GS4-4", "superseded": {"regular": "Standard_GS4"}},
-  "Standard_GS5-8": {"up": "Standard_GS5-16", "down": null, "superseded": {"regular": "Standard_GS5"}},
-  "Standard_GS5-16": {"up": null, "down": "Standard_GS5-8", "superseded": {"regular": "Standard_GS5"}},
-  "Standard_DS11-1_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_DS11_v2"}},
-  "Standard_DS12-1_v2": {"up": "Standard_DS12-2_v2", "down": null, "superseded": {"regular": "Standard_DS12_v2"}},
-  "Standard_DS12-2_v2": {"up": null, "down": "Standard_DS12-1_v2", "superseded": {"regular": "Standard_DS12_v2"}},
-  "Standard_DS13-2_v2": {"up": "Standard_DS13-4_v2", "down": null, "superseded": {"regular": "Standard_DS13_v2"}},
-  "Standard_DS13-4_v2": {"up": null, "down": "Standard_DS13-2_v2", "superseded": {"regular": "Standard_DS13_v2"}},
-  "Standard_DS14-4_v2": {"up": "Standard_DS14-8_v2", "down": null, "superseded": {"regular": "Standard_DS14_v2"}},
-  "Standard_DS14-8_v2": {"up": null, "down": "Standard_DS14-4_v2", "superseded": {"regular": "Standard_DS14_v2"}},
-  "Standard_M416-208s_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_M416s_v2"}},
-  "Standard_M416-208ms_v2": {"up": null, "down": null, "superseded": {"regular": "Standard_M416ms_v2"}},
+  "Standard_M8-2ms": {"up": "Standard_M8-4ms", "down": null, "superseded": null},
+  "Standard_M8-4ms": {"up": null, "down": "Standard_M8-2ms", "superseded": null},
+  "Standard_M16-4ms": {"up": "Standard_M16-8ms", "down": null, "superseded": null},
+  "Standard_M16-8ms": {"up": null, "down": "Standard_M16-4ms", "superseded": null},
+  "Standard_M32-8ms": {"up": "Standard_M32-16ms", "down": null, "superseded": null},
+  "Standard_M32-16ms": {"up": null, "down": "Standard_M32-8ms", "superseded": null},
+  "Standard_M64-16ms": {"up": "Standard_M64-32ms", "down": null, "superseded": null},
+  "Standard_M64-32ms": {"up": null, "down": "Standard_M64-16ms", "superseded": null},
+  "Standard_M128-32ms": {"up": "Standard_M128-64ms", "down": null, "superseded": null},
+  "Standard_M128-64ms": {"up": null, "down": "Standard_M128-32ms", "superseded": null},
+  "Standard_E4-2s_v3": {"up": null, "down": null, "superseded": null},
+  "Standard_E8-2s_v3": {"up": "Standard_E8-4s_v3", "down": null, "superseded": null},
+  "Standard_E8-4s_v3": {"up": null, "down": "Standard_E8-2s_v3", "superseded": null},
+  "Standard_E16-4s_v3": {"up": "Standard_E16-8s_v3", "down": null, "superseded": null},
+  "Standard_E16-8s_v3": {"up": null, "down": "Standard_E16-4s_v3", "superseded": null},
+  "Standard_E32-8s_v3": {"up": "Standard_E32-16s_v3", "down": null, "superseded": null},
+  "Standard_E32-16s_v3": {"up": null, "down": "Standard_E32-8s_v3", "superseded": null},
+  "Standard_E64-16s_v3": {"up": "Standard_E64-32s_v3", "down": null, "superseded": {null},
```

Got an extra squiggly bracket here 
```suggestion
  "Standard_E64-16s_v3": {"up": "Standard_E64-32s_v3", "down": null, "superseded": null},
```

---

Don't ask how long it took me to find this 😆

---

Didn't notice. Thank you so much for that!